ivadomed/config/config_new_loader.json

```diff
@@ -9,18 +9,18 @@
         "safety_factor": [1.0, 1.0, 1.0]
     },
     "loader_parameters": {
-        "path_data": ["data_example_spinegeneric"],
+        "path_data": ["data_example_microscopy_sem"],
         "bids_config": "ivadomed/config/config_bids.json",
         "subject_selection": {"n": [], "metadata": [], "value": []},
-        "target_suffix": ["_seg-manual"],
-        "extensions": [],
+        "target_suffix": ["_seg-axon-manual", "_seg-myelin-manual"],
+        "extensions": [".png"],
```

Comment:
You are correct. I change this config file with every PR for the loader/microscopy project for testing. When it will be completed, I will delete it and add an "official" microscopy config file.